src/material-experimental/mdc-chips/chip-set.ts

```diff
@@ -117,6 +119,9 @@ export class MatChipSet extends _MatChipSetMixinBase implements AfterContentInit
    */
   _mdcClasses: {[key: string]: boolean} = {};
 
+  @Input()
+  chipRemovedMessage = 'Chip removed';
```

Comment:
I'm not sure that the user would know what "chip" refers to in this case since the chips have `role="option"`. Maybe it should just say "removed"?

Reply:
Agreed that "chip" doesn't actually mean anything in a user context. 

I'm not totally convinced that _always_ having a remove message is the right thing to do, or that it should be built into the component vs. letting the user make an announcement based on the remove event. Thoughts?